### ISSUE TITLE
Encoding / accents issues translating back from R

### ISSUE THREAD

**Issue by luispfonseca** [luispfonseca/stata-rcallcountrycode#1]
Doesn't work well with countries with accents like Curaçao, Côte d'Ivoire, etc.

**Comment by luispfonseca** [comment]
0a4ed0a  addresses the issue in Windows, at least, for now.

Instead of using st.load() function, I export in R to a csv file and load that csv file, which is likely to be slower. However, I expect mose use cases to have a limited number of observations (as we only provide R with the list of unique strings), which should minimize the impact.

The issue is still not fully solved. Example:
``` stata
. clear
. input str20 country

                  country
  1. "United Kingdom"
  2. end

. rcallcountrycode country, from(country.name.en) to(country.name.de) gen(de)
Preparing data to send to R
Calling R...
[1] "Using countrycode package version: 1.1.0"
Merging the data

. list

     +---------------------------------+
     |        country              bla |
     |---------------------------------|
  1. | United Kingdom   Gro�britannien |
     +---------------------------------+
```

Also not tested outside windows

**Comment by luispfonseca** [comment]
Using fileEncoding option in R here:
https://github.com/luispfonseca/stata-rcallcountrycode/blob/236bef91fde57cc957472323d4749f7cabca325d/rcallcountrycode.ado#L93

makes it worse.

**Comment by luispfonseca** [comment]
https://github.com/luispfonseca/stata-rcallcountrycode/blob/0a4ed0a22b50b5ef82cb2f3d3c3734950923c21b/rcallcountrycode.ado#L93

Adding fileEncoding option with utf8 as argument to pass to R makes it worse

**Comment by luispfonseca** [comment]
Hopefully addressed in v0.1.3, commit 1027240f9c0e54b53558d9b5c2f99868eb1d7d18. Need to test outside Windows.

Solution was to move data transfer out of `rcall' using csv files to export and import and try to ensure proper encoding. Not sure if the issue is with `rcall' (I tried to see what could work in there, but without success), or with Stata or windows not liking the encodings.

See https://github.com/luispfonseca/stata-rcallcountrycode/compare/a76888c4eecb26f94e98678d44bd112ec27ea5df...4ffc49f for the main changes

**Comment by luispfonseca** [comment]
Tests of diacritics in Linux were fine, both when feeding into R and taking in its output.
